controllers/twophase/networkchaos/netem/types.go

```diff
@@ -0,0 +1,56 @@
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package netem
+
+import (
+	"context"
+
+	"github.com/pingcap/chaos-mesh/controllers/networkchaos/netem"
+
+	"github.com/go-logr/logr"
+
+	"github.com/pingcap/chaos-mesh/controllers/twophase"
+
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// NewTwoPhaseReconciler would Create TwoPhaseReconciler
+func NewTwoPhaseReconciler(c client.Client, log logr.Logger, req ctrl.Request) *TwoPhaseReconciler {
+	r := &netem.Reconciler{
+		Client: c,
+		Log:    log,
+	}
+	return &TwoPhaseReconciler{
+		Reconciler: r,
+	}
+}
+
+// TwoPhaseReconciler reconcile the networkchaos
```

Comment:
It seems two phase pod has the [same comments](https://github.com/pingcap/chaos-mesh/blob/master/controllers/twophase/podfailure/types.go).  A more friendly comment would be `TwoPhaseReconciler implements twophase.InnerObject for netem.Reconciler so it can xxxx` It would be good to have some comment on the [InnerObject](https://github.com/pingcap/chaos-mesh/blob/master/controllers/twophase/types.go#L34) for two phase as well